src/pages/ProductsList/ProductsList.jsx

```diff
@@ -2,30 +2,46 @@ import React, { useState, useEffect } from 'react';
 import { useParams, useSearchParams } from 'react-router-dom';
 import KakaoMap from './components/KakaoMap';
 import SearchBar from '../../components/Nav/SearchBar';
-import SearchInput from './components/SearchInput';
+import SearchCamp from './components/SearchCamp';
 import FilterBox from './components/FilterBox';
 import SortByBox from './components/SortByBox';
 import ImagesContainer from './components/ImagesContainer';
 import KakaoModal from './components/KakaoModal';
+import SiteBanner from './components/SiteBanner';
 import styled from 'styled-components';
 import { flexSort } from '../../styles/mixin';
 
 const ProductsList = () => {
-  const PRODUCT_LIST_API = 'http://10.58.52.241:3000';
+  const PRODUCT_LIST_API = 'http://10.58.52.176:3000';
   const [product, setProduct] = useState([]);
-  const [searchParams, setSearchParams] = useSearchParams([]);
+  const [searchParams, setSearchParams] = useSearchParams(
+    new URLSearchParams()
+  );
```

Comment:
```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
```
해당 부분에 새로운 searchParams를 적용하지 않아도 됩니다!